### PR TITLE
button and topbar ui updates

### PR DESCRIPTION
**Pull request by pinkhoodie**
#### CheckList
- [x] Updated button fontweight
- [x] Removed sub header of topbar's background
- [x] Shortened width of inputs in wide screen and removed borderBottom

### CODE REVIEW

**Review by tetogomez — APPROVED** [review]
LGTM

File: src/routes/Evodex/BackLayer/Exchange/ExchangeBackLayer.js [hunk 1]
```diff
@@ -88,6 +88,10 @@ const useStyles = makeStyles((theme) => ({
     alignItems: 'center',
     marginTop: theme.spacing(1),
     padding: theme.spacing(2, 0),
+    width: '70%',
```
Comment (xavier506) [inline src/routes/Evodex/BackLayer/Exchange/ExchangeBackLayer.js:91]: why reduce the width to 70% ? 

The exchange page looks off center and is not consistent 
image

https://deploy-preview-151--evodex.netlify.app/exchange

**Review by xavier506 — CHANGES_REQUESTED** [review]
please see comments and resolve merge conflicts

File: src/components/InputTextAndSelect/InputTextAndSelect.js [hunk 2]
```diff
@@ -73,7 +74,7 @@ const useStyles = makeStyles((theme) => ({
     backgroundColor: 'rgba(255,255,255,0.05)',
     borderTopLeftRadius: 4,
     borderTopRightRadius: 4,
-    borderBottom: '2px solid rgba(255,255,255,0.38)',
+    borderBottom: '0px solid rgba(255,255,255,0.38)',
```
Comment (xavier506) [inline src/components/InputTextAndSelect/InputTextAndSelect.js:77]: The bottom border was in the original wireframe, why should we remove it?

**Pull request closed without merging**